src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -86,8 +86,7 @@ TxQ::FeeMetrics::update(Application& app,
     feeLevels.reserve(txnsExpected_);
     for (auto const& tx : view.txs)
     {
-        auto const baseFee = calculateBaseFee(app, view,
-            *tx.first, j_);
+        auto const baseFee = calculateBaseFee(view, *tx.first);
```

Comment:
`app` no longer appears needed in this function.

Reply:
It is not, this change is removing `app` and `j_`.